library/src/main/java/com/bumptech/glide/util/LruCache.java

```diff
@@ -119,17 +119,16 @@ public synchronized Y get(T key) {
    * @param key  The key to add the item at.
    * @param item The item to add.
    */
-  public synchronized Y put(T key, @Nullable Y item) {
+  @Nullable
+  public synchronized Y put(@NonNull T key, @NonNull Y item) {
```

Comment:
The `item` nullability was incorrect: `getSize` would blow on null so it can't be null (also didn't notice anyone in our codebase using null).

Reply:
Hmmm, the tests have told me that's wrong. Is 63a73de correct?